erica_app/Development.Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.9.10-slim-buster
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+ARG ERICA_BUCKET_NAME
+ARG AWS_ACCESS_KEY_ID
+ARG AWS_SECRET_ACCESS_KEY
+ARG ENDPOINT_URL
+ARG ELSTER_DATENLIEFERANT
+ARG ELSTER_HERSTELLER_ID
+ENV ERICA_BUCKET_NAME=$ERICA_BUCKET_NAME
+ENV AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+ENV AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+ENV ENDPOINT_URL=$ENDPOINT_URL
```

Comment:
Why do we set them here and not just use them similar to line 32. We do not need them es env variables in the image and probably do not want them there either. This would make it harder for devs to keep track of where the secrets are stored. wdyt?

Reply:
But I can move 32 to line 18

Reply:
I would not want to set the arguments that are only needed to install the Erica binaries (only during build) in the environment of the image because this makes it very hard to know where the secrets are. Therefore we could omit that here and use something like ```RUN env AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY  ENDPOINT_URL=$ENDPOINT_URL python scripts/load_eric_binaries.py download-eric-cert-and-binaries``` in line 31. Does that make sense?

Reply:
We should add the binaries and constraints in the build step. Docker run is not for provisioning the container, it is only for running.